cmake/ActsExternSources.cmake

```diff
@@ -31,7 +31,7 @@ set( ACTS_FRNN_SOURCE
 mark_as_advanced( ACTS_FRNN_SOURCE )
 
 set( ACTS_GEOMODEL_SOURCE
-   "GIT_REPOSITORY;https://gitlab.cern.ch/GeoModelDev/GeoModel;GIT_TAG;4.6.0;" CACHE STRING "Source to take GeoModel from")
+   "GIT_REPOSITORY;https://gitlab.cern.ch/GeoModelDev/GeoModel;GIT_TAG;5.3.0;" CACHE STRING "Source to take GeoModel from")
```

Comment:
Will 6.2.0 require additional changes?

Reply:
This I will need to check. I think I will make this PR draft until I see what is happening with 6.2.0